torch/onnx/symbolic_opset11.py

```diff
@@ -44,9 +43,32 @@ def pixel_shuffle(g, self, upscale_factor):
         return _unimplemented("pixel_shuffle", "only support 4d input")
     return g.op("DepthToSpace", self, blocksize_i=upscale_factor, mode_s="CRD")
 
+def interpolate_quantized(g, input, output_size, dim, interpolate_mode, align_corners=None):
```

Comment:
Do we have quantized upsample op in ONNX? I think in ONNX it is just a regular operator which supports uint8 as inputs. So we can just call interpolate symbolic in interpolate_quantized.

Reply:
The issue with upsample is that `input_size = input.type().sizes()` translates to None for quantized input. So for now I've set input size to a tensor of size 1 in interpolate_quantized and separated it out. We will have to handle it in the onnx->c2 conversion flow.